cmd/skopeo/copy.go

```diff
@@ -74,7 +74,7 @@ func copyHandler(context *cli.Context) error {
 	}
 
 	// FIXME: We need to call PutManifest after PutBlob and PutSignatures. This seems ugly; move to a "set properties" + "commit" model?
-	if err := dest.PutManifest(manifest); err != nil {
+	if err := dest.PutManifest(m); err != nil {
```

Comment:
The 3 `cmd/skopeo/copy.go` hunks can be dropped: after #103 `copy.go` does not import `skopeo/manifest`, so we can use the more descriptive `manifest` variable name.